Snippets/ASB/ASB_7/Usage.cs

```diff
@@ -78,5 +79,17 @@ void TopologySelectionUpgradeGuide(EndpointConfiguration endpointConfiguration)
 
             #endregion
         }
+
+        void AutoLockRenewal(EndpointConfiguration endpointConfiguration)
+        {
+            var maximumProcessingTime = TimeSpan.FromMinutes(5);
+
+            #region asb-auto-lock-renewal
+
+            endpointConfiguration.UseTransport<AzureServiceBusTransport>()
+                .MessageReceivers().AutoRenewTimeout(maximumProcessingTime);
```

Comment:
Not sure about that. IDE will provide that information. Here, the concept is more important than the type.
@SimonCropp if you can find a strong argument in favour of using `TimeSpan`, I'm listening :)